doc/nvim-treesitter.txt

```diff
@@ -175,71 +178,52 @@ Supported options:
   }
   EOF
 <
-==============================================================================
-USER QUERY EXTENSIONS                        *nvim-treesitter-query-extensions*
-
-Queries are what `nvim-treesitter` uses to extract informations from the syntax tree, and they are
-located in the `queries/{lang}/*` runtime directories (like the `queries` folder of this plugin).
-
-`nvim-treesitter` considers queries as any runtime file (see `:h rtp`), that is :
-
-- if the file is in any `after/queries/` folder, then it will be used to extend the already defined
-  queries.
-- Otherwise, it will be used as a base to define the query, the first query found (with the highest
-  priority) will be the only one to be used.
-
-This hybrid approach is the most standard way, and according to that, here is some ideas on how to
-use is :
-- If you want to rewrite (or write) a query, don't use `after/queries`.
-- If you want to override a part of a query (only one match for example), use the `after/queries`
-  directory.
-
 ==============================================================================
 COMMANDS                                            *nvim-treesitter-commands*
 
-                                                                  *:TSInstall*
+								  *:TSInstall*
```

Comment:
I'm more fan of spaces, but all good p:

Reply:
Default help style is with tabs, so we go for tabs.